package.json

```diff
@@ -32,20 +32,19 @@
   },
   "homepage": "https://github.com/emiljanitzek/mongoose-delete-ts",
   "peerDependencies": {
-    "mongoose": "6.x"
+    "mongoose": "8.x"
   },
   "devDependencies": {
-    "@tsconfig/node18": "~1.0.1",
-    "@types/chai": "~4.3.1",
-    "@types/mocha": "~9.1.1",
-    "@typescript-eslint/eslint-plugin": "~5.30.6",
-    "@typescript-eslint/parser": "~5.30.6",
-    "chai": "~4.3.6",
-    "eslint": "~8.19.0",
-    "mocha": "~10.0.0",
-    "mongoose": "~6.5.2",
-    "ts-mocha": "~10.0.0",
```

Comment:
There is a PR to update dependencies to the latest and make `ts-node` to peer dependency in ts-mocha.
Once it's merged and new version is released, we will use ts-mocha again.
At the moment `ts-node` version of `ts-mocha` is too old.